### ISSUE TITLE
Clean up a few things to get rid of Warnings

### ISSUE THREAD

**Issue by andrew-edwards** [pbs-assess/pacea#56]
From R-CMD-check (online, linked from checkmark next to commit), fix some Warnings:

-[ ] Warning: '::' or ':::' import not declared from: ‘tibble’
-[ ] Warning: Invalid license file pointers: LICENSE

Undocumented data sets:
-[ ] grid26    - @travistai2 can you do this one, I'll do the rest.
-[ ] soi
-[X] test_grid_20

Recent commit should fix this one, just a missing s in clim_anom() help:
-[ ] clim_years
-[ ] clim_year  [not in \usage]

plot_with_uncertainty_discrete:
-[ ] y_tick undocumented

**Comment by travistai2** [comment]
I'm not sure if 'test_grid_20' was used anywhere - I didn't use it.

**Comment by andrew-edwards** [comment]
Yes, I think I got rid of it (hence the box was ticked). Sorry, that wasn't obvious.

**Comment by andrew-edwards** [comment]
- [ ] Also this one https://github.com/pbs-assess/pacea/actions/runs/8841447116/job/24278561740#step:5:98 about adding `...` to `plot.XXX()` functions. We hadn't included them because they call ggplot functions not base plot. However, I think it's okay to add them in (but not pass them onto anything), as as is done in ggplot itself (`?ggplot`). Trying one first to see if it changes the warning, but the warning isn't really specific about a particular function.

**Comment by andrew-edwards** [comment]
Warnings: 
- [ ] Variables with usage in Rd file `bccm_eez_poly.Rd` but not in code: `bccm_hotssea_poly`
- [ ] figure out if I need to commit any more .rda files (or just add to .gitignore)
- [ ] Undocumented code objects:  'grid_hotssea' 'roms_buff' 'roms_cave' 'soi' 'surf_dat'  'surf_hotssea_buff'
- [ ] Undocumented data sets: 'grid_hotssea' 'roms_buff' 'roms_cave' 'soi' 'surf_dat' 'surf_hotssea_buff'
- [ ]  checking S3 generic/method consistency ... WARNING
  ```
plot:
    function(x, ...)
plot.pacea_buoy:
    function(obj, stn_id, years, year_highlight, ...)
  ``` and more; think I just need to change `obj` to `x` in them (all!).
